desktop/lib/messagebundles/it.po

```diff
@@ -217,12 +217,12 @@ msgstr "Invia a un amico"
 #: /Users/gubatron/workspace.frostwire/frostwire/desktop/src/com/frostwire/gui/bittorrent/BTDownloadActions.java:385
 #: /Users/gubatron/workspace.frostwire/frostwire/desktop/src/com/frostwire/gui/library/AbstractLibraryTableMediator.java:219
 msgid "Create New Playlist"
-msgstr "Crea nuova Playlist"
+msgstr "Crea nuova lista di riproduzione"
```

Comment:
this won't fit, anyway to make it shorter?

Reply:
try `gradle gettextBundle` and then run frostwire so you see how it looks on the Library window, on the left hand side where you can create playlists.

Reply:
So the option we have is to continue using "playlist"

Reply:
Do you prefer playlist instead of lista di riproduzione?

Reply:
In this instances I think it makes more sense, but then, if you do it here, you should probably do it everywhere else for consistency.